symbolicationWebService.py

```diff
@@ -199,7 +201,7 @@ def Main():
   if not ReadConfigFile():
     return 1
 
-  SetTracingEnabled(gOptions["enableTracing"] > 0)
+  SetTracingEnabled(gOptions["enableTracing"] > 0, gOptions["Log"])
```

Comment:
same as before, couldn't you rely on the logging library's log-level instead of maintaining gEnableTracing